server.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const port = process.env.PUBLIC_PORT ;
```

Comment:
It's a good practice to validate environment variables before using them. Consider checking if the port is defined and is a number before starting the server. [medium]